owlkettle/widgets.nim

```diff
@@ -3525,6 +3525,79 @@ renderable Scale of BaseWidget:
         echo "New value is ", newValue
         app.value = newValue
 
+renderable Expander of BaseWidget:
+  ## Wrapper for GTK Expander Widget: https://docs.gtk.org/gtk4/class.Expander.html
```

Comment:
Usually this is a summary of what the widget does & does not include a link to the GTK docs.
```suggestion
  ## Container that shows or hides its child depending on whether it is expanded/collapsed 
```

Reply:
Fair. I had written that one before I saw you preferred not to have direct gtk doc links.
To me it just seemed sensible that users should have a link to the more expansive docs for details about the Widget, thus I had included them there. 
Spare them the necessity of a google search where they might have bad googling skills and not find the right docs and all that.